src/components/BShuttle/BShuttle.vue

```diff
@@ -0,0 +1,480 @@
+        id='all-to-available'
+        type='button'
+        >
+        <img class="icon" :src="allToAvailable" />
+      </button>
+    </div>
+
+    <div class='chosen'>
+      <label id='chosen-label'>{{ chosenLabel }}</label>
+      <ul
+        aria-labeledby='chosen-label'
+        role='selectbox'
+      >
+        <li
+          v-for="option in chosenOptions"
+          :key="'chosen-option-' + id + '-' + option.id"
+          :id="'chosen-option-' + id + '-' + option.id"
+          :class="optionsClass(option, 'chosen')"
+          @click="onOptionClick(option, 'chosen', 'available', $event)"
+          @keyup.space.stop.prevent.stop="onOptionClick(option, 'chosen', 'available', $event)"
+          @keyup.down.prevent.stop="onKeyupDown($event, option, 'chosen', 'available')"
+          @keyup.up.prevent.stop="onKeyupUp($event, option, 'chosen', 'available')"
+          @keyup.left.prevent.stop="onMoveSelectedClick('chosen')"
+          @keyup.enter.prevent.stop="onMoveSelectedClick('chosen')"
+          @keydown.up.prevent.stop
+          @keydown.down.prevent.stop
+          @keydown.space.prevent.stop
+          role='option'
+          tabindex='0'
+        >
+          {{ option.displayText }} 
+
+        </li>
+      <span v-show="errors.has(validationId)" class="error-text" :id="'error-' + id">{{ errors.first(validationId) }}</span>
+      </ul>
+
+    </div>
+
+  </div>
+  
+</template>
+
+<script>
+
+import validationMixIn from '../../mixins/validation'
+
+export default {
+  name: 'b-shuttle',
+  mixins: [validationMixIn()],
+  props: {
+    /**
+    * The available options for the shuttle.
+    */
+    options: {
+      type: Array,
+      required: false
+    },
+    /**
+    * Placeholder to display if shuttle is dependent on other component.
+    */
+    placeholder: {
+      type: String,
+      required: false
+    },
+    /**
+    * Label to display above available options.
+    */
+    availableLabel: {
+      type: String,
+      required: true
+    },
+    /**
+    * Label to display above chosen options.
+    */
+    chosenLabel: {
+      type: String,
+      required: true
+    },
+    /**
+    * Use to turn on validation
+    */
+    required: {
+      type: Boolean,
+    },
+    /**
+    * The id(s) of the selected option(s).
+    */
+    value: {
+      type: Array,
+      required: false,
+      default: function () { return [] }
+    }
+  },
+
+  data () {
+    return {
+      availableOptions: (this.options && this.options.length > 0) ? this.setOptions(this.options, this.value, 'available') : new Array,
+      selectedOptions: {'available': new Array, 'chosen': new Array},
+      chosenOptions: (this.value && this.options) ? this.setOptions(this.options, this.value, 'chosen') : new Array,
+      lastClick: 0,
+      id: null,
+      allToChosen: require('../../../static/doubleArrow.svg'),
+      allToAvailable: require('../../../static/doubleArrow-left.svg')
+    }
+  },
+
+  computed: {
```

Comment:
Don't need this.

Reply:
sorry, that was a refactor leftover, will remove